src/taxes/pages/TaxChannelsPage/TaxChannelsPage.tsx

```diff
@@ -142,6 +143,21 @@ export const TaxChannelsPage: React.FC<TaxChannelsPageProps> = props => {
           set({ updateCountriesConfiguration: currentExceptions });
         };
 
+        const handleCountryChange = (country: CountryFragment) => {
+          closeDialog();
+          const input = {
+            country,
+            chargeTaxes: data.chargeTaxes,
+            displayGrossPrices: data.displayGrossPrices,
+            taxCalculationStrategy: data.taxCalculationStrategy,
+          } as TaxConfigurationPerCountryFragment;
```

Comment:
if you can't fulfill the required type, don't force it, perhaps you need to add all required properties, of use Pick a new type

suggestion: try to infer that

Reply:
I think I'm gonna like the upcoming `satisfies` operator a lot :v
Here we can actually just add __typename and we can use this type normally